python/doc/examples/probabilistic_modeling/distributions/plot_posterior_distribution.py

```diff
@@ -0,0 +1,480 @@
+c.setColor("blue")
+c.setLineStyle("dashed")
+g_ML.add(c)
+g_ML.setLegends(
+    [
+        r"dist of $\mathbf{\boldsymbol{\theta}}_n^{MV}$",
+        r"$\mathbf{\boldsymbol{\theta}}_n^{MV}$",
+        r"$g(\mathbf{\boldsymbol{Y}}_n^m)$",
+        "ML CI(" + str(int(100 * alpha)) + "%)",
+        "Bay CI(" + str(int(100 * alpha)) + "%)",
+    ]
+)
+g_ML.setXTitle(r"$\sigma_0$")
+g_ML.setYTitle(r"$\sigma_1$")
+g_ML.setTitle("ML Estimator")
+
+# %%
+# Then the Bayesian estimator.
+g_Bay = dist_estimateur_Bay.drawPDF([0.5] * 2, [1.5] * 2)
+c = ot.Cloud([theta_Bay[[1, 3]]])
+c.setColor("red")
+c.setPointStyle("bullet")
+g_Bay.add(c)
+c = ot.Cloud([theta_ML[[1, 3]]])
+c.setColor("red")
+c.setPointStyle("+")
+g_Bay.add(c)
+a = interval_Bay.getLowerBound()
+b = interval_Bay.getUpperBound()
+c = ot.Curve([a, [a[0], b[1]], b, [b[0], a[1]], a])
+c.setColor("blue")
+g_Bay.add(c)
+a = interval_ML.getLowerBound()
+b = interval_ML.getUpperBound()
+c = ot.Curve([a, [a[0], b[1]], b, [b[0], a[1]], a])
+c.setColor("blue")
+c.setLineStyle("dashed")
+g_Bay.add(c)
+g_Bay.setLegends(
+    [
+        r"dist of $g(\pi_{n, \mathbf{\boldsymbol{Y}}})$",
+        r"$g(\mathbf{\boldsymbol{Y}}_n^m)$",
+        r"$\mathbf{\boldsymbol{\theta}}_n^{MV}$",
+        "Bay CI(" + str(int(100 * alpha)) + "%)",
+        "ML CI(" + str(int(100 * alpha)) + "%)",
+    ]
+)
+g_Bay.setXTitle(r"$\sigma_0$")
+g.setYTitle(r"$\sigma_1$")
+g_Bay.setTitle("Bayesian Estimator")
+
+# Gather both graph in a grid.
+grid = ot.GridLayout(1, 2)
+grid.setGraph(0, 0, g_ML)
+grid.setGraph(0, 1, g_Bay)
+
+view = otv.View(grid)
+
+# %%
+# Finally, the following table sums up the previous computed quantities.
+#
+# ============     ========================  ========================  ======================   ======================
+# Approach         :math:`\tilde{\sigma}_0`  :math:`\tilde{\sigma}_1`  IC(:math:`\sigma_0`)     IC(:math:`\sigma_1`)
+# ============     ========================  ========================  ======================   ======================
+# Frequentist      0.967                     0.951                     :math:`[0.651, 1.30]`    :math:`[0.659, 1.20]`
+# Bayesian         0.950                     0.934                     :math:`[0.736, 1.38]`    :math:`[0.724, 1.35]`
+# ============     ========================  ========================  ======================   ======================
+#
+# ============     =============   =======================  =================================================
+# Approach         :math:`\beta`   :math:`KL`               :math:`\cV(IC(\tilde{\sigma}_0, \tilde{\sigma}))`
+# ============     =============   =======================  =================================================
+# Frequentist      0.974           :math:`3.46\, 10^{-3}`   0.352
+# Bayesian         0.974           :math:`6.87\, 10^{-3}`   0.381
+# ============     =============   =======================  =================================================
+#
+# %%
```

Comment:
I think it wants an empty line between the two cells, here the comment goes on to the next cell